.buildkite/pipeline.yml

```diff
@@ -1,5 +1,14 @@
 steps:
 
+  #
+  # License audit
+  #
+  - label: ':copyright: License Audit'
+    timeout_in_minutes: 20
+    agents:
+      queue: opensource-mac-cocoa-11
```

Comment:
why does this need to run on a mac?

Reply:
Not as such, I don't think - but I had problems running the license_finder tool in its Docker image, either on CI or locally.  It yielded different results, so I chose to go for one of our Macs as I had the most confidence in it working.